providers/base/units/rtc/jobs.pxu

```diff
@@ -14,3 +14,91 @@ user: root
 category_id: rtc_category
 estimated_duration: 40
 flags: noreturn
+
+id: rtc/rtc_number
+_summary: Check the number of RTC as expected.
+_description:
+    Check the number of RTC on platform as expected.
+plugin: shell
+user: root
+estimated_duration: 1.0
+category_id: com.canonical.certification::rtc_category
+command:
+    if [ -z "$TOTAL_RTC_NUM" ]; then 
```

Comment:
If this is coming from the Checkbox launcher/conf `[environment]` section, it's better to add a

```
environ:  TOTAL_RTC_NUM
```

section in the job, otherwise the job may not run properly in remote mode.

Also, just to be safe, I would add an `echo TOTAL_RTC_NUM not defined, defaulting to 1` here, so that if there is something wrong with Checkbox or the config, it's easier to debug down the line.